Added borderless icon class

Blackbaud-AlexKingman opened the pull request.

blackbaud-ado commented:
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

blackbaud-ado commented:
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

Blackbaud-SteveBrush commented on src/app/public/styles/themes/modern/_buttons.scss:
```diff
@@ -77,6 +77,16 @@
 
+  .sky-btn-icon-borderless {
+    @include sky-theme-modern-button-variant(
+      $sky-theme-modern-font-deemphasized-color,
```
Is this color correct? It's barely perceptible in the visual tests.

Blackbaud-ToddRoberts replied:
the default theme version is super light, that's why we darkened it for modern theme. it should be the normal deemphasized color, which looks like it is to me

Blackbaud-ToddRoberts replied:
tho maybe i am looking at the wrong button?

Blackbaud-AlexKingman commented:
Hold: Waiting on specs for borderless icon class.

blackbaud-ado commented:
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

Blackbaud-AlexKingman commented:
@Blackbaud-SteveBrush @Blackbaud-ToddRoberts 
Updated for new design spec: Borderless buttons should be 26x26. Also added support for default mode.

This is ready for review again.

blackbaud-ado commented:
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)
